receiver/elasticapmintakereceiver/receiver_test.go

```diff
@@ -510,6 +510,6 @@ func runComparisonForMetrics(t *testing.T, inputJsonFileName string, expectedYam
 	// golden.WriteMetrics(t, expectedFile, actualMetrics)
 	expectedMetrics, err := golden.ReadMetrics(expectedFile)
 	require.NoError(t, err)
-	require.NoError(t, pmetrictest.CompareMetrics(expectedMetrics, actualMetrics, pmetrictest.IgnoreMetricsOrder()))
+	require.NoError(t, pmetrictest.CompareMetrics(expectedMetrics, actualMetrics, pmetrictest.IgnoreMetricsOrder(), pmetrictest.IgnoreTimestamp()))
```

Comment:
why pmetrictest.IgnoreTimestamp() was added?

Reply:
`golden.WriteMetrics` changed the expected timestamps to a default value of `100000`, breaking the tests and we also ignore the timestamps even for traces. However, it was cheap to reinstate the previous changes (see https://github.com/elastic/opentelemetry-collector-components/compare/278caaebf7d6683fad32433b96358ced5a76e907..cd715b03c47f0a0f7d59eec404fcca55ba6774de)